lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -1536,7 +1534,7 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
   }
 };
 
-class ReferencedTypeFinder : private TypeVisitor<ReferencedTypeFinder> {
+class ReferencedTypeFinder : public TypeVisitor<ReferencedTypeFinder> {
```

Comment:
I was going to say "why not just redeclare the `visit` method" but I guess it doesn't matter; these are all local helper types anyway.